ui/package-lock.json

```diff
@@ -0,0 +1,5892 @@
+{
```

Comment:
❗ 😬 Ahhhh npm was use to pull deps

Reply:
What should be used?

Reply:
`yarn` it's basically npm but easier and cleaner.

The global install is  `npm install --global yarn`

Reply:
I ran that yarn command and it change anything. What am I missing?

Reply:
You just gotta remove the packages-lock.json.

Reply:
Ok, did that. Does it look right now?